Library/Homebrew/exceptions.rb

```diff
@@ -490,8 +490,8 @@ def issues
   sig { returns(T::Array[T.untyped]) }
   def fetch_issues
     GitHub.issues_for_formula(formula.name, tap: formula.tap, state: "open", type: "issue")
-  rescue GitHub::API::RateLimitExceededError => e
-    opoo e.message
+  rescue GitHub::API::Error => e
+    opoo "Unable to query GitHub for recent issues on the tap\n#{e.message}"
```

Comment:
Added text to the beginning of the error message to explain that the error isn't related to the original exception